pkg/ottl/ottlfuncs/README.md

```diff
@@ -1792,9 +1792,21 @@ The `Split` Converter separates a string by the delimiter, and returns an array
 
 If the `target` is not a string or does not exist, the `Split` Converter will return an error.
 
+### Trim
+
+```Trim(target, Optional[replacement])```
+
+The `Trim` Converter removes the leading and trailing character (default: whitespace).
+
+If the `target` is not a string or does not exist, the `Trim` Converter will return an error.
+
+`target` is a string.
+`replacement` is an optional string representing the character to replace with (default: whitespace).
```

Comment:
```suggestion
The `Trim` Converter removes the leading and trailing character (default: a space character).

If the `target` is not a string or does not exist, the `Trim` Converter will return an error.

`target` is a string.
`replacement` is an optional string representing the character to replace with (default: a space character).
```
"Whitespace" often also refers to other non-visible characters like tabs, newlines, etc. We should avoid giving that impression here.